src/components/side-panel.js

```diff
@@ -212,29 +212,56 @@ export default function SidePanelFactory(
-                layerOrder={layerOrder}
-                layerBlending={layerBlending}
-                mapStyle={mapStyle}
-                mapStyleActions={mapStyleActions}
-                mapStateActions={mapStateActions}
-                interactionConfig={interactionConfig}
-                removeDataset={onRemoveDataset}
-                showDatasetTable={onShowDatasetTable}
-                updateTableColor={onUpdateTableColor}
-                showAddDataModal={onShowAddDataModal}
-                showAddMapStyleModal={onShowAddMapStyleModal}
-                uiStateActions={uiStateActions}
-                visStateActions={visStateActions}
-              />
+              currentPanel.id !== 'layer' ? (
+                <>
+                  <PanelTitle className="side-panel__content__title">
+                    <FormattedMessage id={currentPanel.label} />
+                  </PanelTitle>
+                  <PanelComponent
+                    datasets={datasets}
+                    filters={filters}
+                    layers={layers}
+                    layerClasses={layerClasses}
+                    layerOrder={layerOrder}
+                    layerBlending={layerBlending}
+                    mapStyle={mapStyle}
+                    mapStyleActions={mapStyleActions}
+                    mapStateActions={mapStateActions}
+                    interactionConfig={interactionConfig}
+                    removeDataset={onRemoveDataset}
+                    showDatasetTable={onShowDatasetTable}
+                    updateTableColor={onUpdateTableColor}
+                    showAddDataModal={onShowAddDataModal}
+                    showAddMapStyleModal={onShowAddMapStyleModal}
+                    uiStateActions={uiStateActions}
+                    visStateActions={visStateActions}
+                    panelMetadata={currentPanel}
+                  />
+                </>
+              ) : (
+                <PanelComponent
+                  datasets={datasets}
+                  filters={filters}
+                  layers={layers}
+                  layerClasses={layerClasses}
+                  layerOrder={layerOrder}
+                  layerBlending={layerBlending}
+                  mapStyle={mapStyle}
+                  mapStyleActions={mapStyleActions}
+                  mapStateActions={mapStateActions}
+                  interactionConfig={interactionConfig}
+                  removeDataset={onRemoveDataset}
+                  showDatasetTable={onShowDatasetTable}
+                  updateTableColor={onUpdateTableColor}
+                  showAddDataModal={onShowAddDataModal}
+                  showAddMapStyleModal={onShowAddMapStyleModal}
+                  uiStateActions={uiStateActions}
+                  visStateActions={visStateActions}
+                  panelMetadata={currentPanel}
+                  layerPanelListView={uiState.layerPanelListView}
+                />
+              )
```

Comment:
all of this can be simplified to the following:
```
{currentPanel.id !== 'layer' ? (
    <PanelTitle className="side-panel__content__title">
      <FormattedMessage id={currentPanel.label} />
    </PanelTitle>
    ) : null
  }
  <PanelComponent
    datasets={datasets}
    filters={filters}
    layers={layers}
    layerClasses={layerClasses}
    layerOrder={layerOrder}
    layerBlending={layerBlending}
    mapStyle={mapStyle}
    mapStyleActions={mapStyleActions}
    mapStateActions={mapStateActions}
    interactionConfig={interactionConfig}
    removeDataset={onRemoveDataset}
    showDatasetTable={onShowDatasetTable}
    updateTableColor={onUpdateTableColor}
    showAddDataModal={onShowAddDataModal}
    showAddMapStyleModal={onShowAddMapStyleModal}
    uiStateActions={uiStateActions}
    visStateActions={visStateActions}
    panelMetadata={currentPanel}
    layerPanelListView={currentPanel.id === 'layer' && uiState.layerPanelListView}
  />
```